content/docs/1.5.4/monitoring/prometheus-and-grafana-setup.md

```diff
@@ -61,9 +63,26 @@ Follow instructions in [Prometheus Operator - Quickstart](https://github.com/pro
       - port: manager
     ```
 
-    Longhorn ServiceMonitor has a label selector `app: longhorn-manager` for selecting Longhorn backend service.
+#### Install Longhorn ServiceMonitor with Helm
+
+1. Modify the YAML file `longhorn/chart/values.yaml`.
+
+    ```yaml
+    metrics:
+      serviceMonitor:
+        # -- Setting that allows the creation of a Prometheus ServiceMonitor resource for Longhorn Manager components.
+        enabled: true
+    ```
+
+1. Create a ServiceMonitor for Longhorn Manager using Helm.
+
+    ```bash
+      helm upgrade longhorn longhorn/longhorn --namespace longhorn-system -f values.yaml
+    ```
 
-    Longhorn ServiceMonitor is included in the Prometheus custom resource so that the Prometheus server can discover all Longhorn manager pods and their endpoints.
+Longhorn ServiceMonitor is included in the Prometheus custom resource. This setup allows the Prometheus server to discover all Longhorn Manager pods and their respective endpoints.
```

Comment:
```suggestion
Longhorn ServiceMonitor is a [Prometheus Operator](https://prometheus-operator.dev/) custom resource. This setup allows the Prometheus server to discover all Longhorn Manager pods and their respective endpoints.
```

Reply:
Modified with 1.5.4, 1.6.0/1, and 1.7.0. Thanks.